python/ray/air/_internal/usage.py

```diff
@@ -115,3 +156,25 @@ def tag_scheduler(scheduler: "TrialScheduler"):
     assert isinstance(scheduler, TrialScheduler)
     scheduler_name = _find_class_name(scheduler, "ray.tune.schedulers", TUNE_SCHEDULERS)
     record_extra_usage_tag(TagKey.TUNE_SCHEDULER, scheduler_name)
+
+
+def tag_env_vars() -> bool:
+    """Records environment variable usage.
+
+    Returns:
+        bool: True if at least one environment var is supplied by the user.
+    """
+    all_env_vars = sorted(set().union(AIR_ENV_VARS, TUNE_ENV_VARS))
+
+    user_supplied_env_vars = []
+
+    for env_var in all_env_vars:
+        if os.environ.get(env_var):
```

Comment:
Do you know if there's any correctness benefit with doing this over just checking for existence?
```suggestion
        if env_var in os.environ:
```

Reply:
Ah, I think your proposal is better.

Technically possible to set ENV_VAR="", then my current logic will treat it as not set.